docs/ops/deployment/mesos.zh.md

```diff
@@ -177,16 +177,16 @@ In contrast to that, the task managers will be run as Mesos tasks in the Mesos c
 
 ### Flink job cluster on Mesos
 
-A Flink job cluster is a dedicated cluster which runs a single job.
+A Flink job cluster is a dedicated cluster which runs a single job. 
```

Comment:
This is not a real change. There are just added trailing whitespaces which should be avoided.

Reply:
ok